etna/transforms/encoders/categorical.py

```diff
@@ -0,0 +1,229 @@
+            - If "none", then replace missing values with None
+        """
+        self.in_column = in_column
+        self.inplace = inplace
+        self.strategy = strategy
+        self.out_column = out_column
+        self.out_column = self._get_column_name()
+        super().__init__(transform=_OneSegmentLabelEncoderTransform(self.in_column, self.out_column, self.strategy))
+
+    def _get_column_name(self) -> str:
+        """Get the `out_column` depending on the transform's parameters."""
+        if self.inplace and self.out_column:
+            warnings.warn("Transformation will be applied inplace, out_column param will be ignored")
+        if self.inplace:
+            return self.in_column
+        if self.out_column:
+            return self.out_column
+        if self.in_column.startswith("regressor"):
+            return f"regressor_{self.__repr__()}"
+        return self.__repr__()
+
+
+class _OneSegmentOneHotEncoderTransform(Transform):
+    """Create one-hot encoding columns."""
+
+    def __init__(self, in_column: str, out_column: str):
+        """
+        Create instance of _OneSegmentOneHotEncoderTransform.
+
+        Parameters
+        ----------
+        in_column:
+            name of column to apply transform to
+        out_column:
+            name of added column
+        """
+        self.in_column = in_column
+        self.out_column = out_column
+        self.ohe = preprocessing.OneHotEncoder(handle_unknown="ignore", sparse=False)
+
+    def fit(self, df: pd.DataFrame) -> "_OneSegmentOneHotEncoderTransform":
+        """
+        Fit One Hot encoder.
+
+        Parameters
+        ----------
+        df:
+            dataframe with data to fit the transform.
+        Returns
+        -------
+        self
+        """
+        self.ohe.fit(np.array(df[self.in_column]).reshape(-1, 1))
+        return self
+
+    def transform(self, df: pd.DataFrame) -> pd.DataFrame:
+        """
+        Encode the `in_column` by fitted Label Binarize encoder.
+
+        Parameters
+        ----------
+        df
+            dataframe with data to transform.
+        Returns
+        -------
+        result dataframe
+        """
+        result_df = df.copy()
+        result_df[[self.out_column + "_" + str(i) for i in range(len(self.ohe.categories_[0]))]] = self.ohe.transform(
+            np.array(df[self.in_column]).reshape(-1, 1)
+        )
+        result_df[[self.out_column + "_" + str(i) for i in range(len(self.ohe.categories_[0]))]] = result_df[
+            [self.out_column + "_" + str(i) for i in range(len(self.ohe.categories_[0]))]
+        ].astype("category")
+        return result_df
+
+
+class OneHotEncoderTransform(PerSegmentWrapper):
+    """
+    Encode categorical feature as a one-hot numeric features.
+    If unknown category is encountered during transform, the resulting one-hot encoded columns for this feature will be all zeros.
```

Comment:
It seems like we have to add empty line after the first line with short description: [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard).